zkevm-circuits/src/evm_circuit.rs

```diff
@@ -158,14 +158,14 @@ pub(crate) enum FixedLookup {
 
 impl<F: FieldExt> Expr<F> for FixedLookup {
     fn expr(&self) -> Expression<F> {
-        Expression::Constant(F::from_u64(*self as u64))
+        Expression::Constant(F::from(*self as u64))
     }
 }
 
 #[derive(Clone, Debug)]
 pub(crate) enum Lookup<F> {
-    FixedLookup(FixedLookup, [Expression<F>; 3]),
-    BusMappingLookup(BusMappingLookup<F>),
+    FixedlookupAny(FixedLookup, [Expression<F>; 3]),
+    BusMappinglookupAny(BusMappingLookup<F>),
```

Comment:
This should not be renamed
```suggestion
    FixedLookup(FixedLookup, [Expression<F>; 3]),
    BusMappingLookup(BusMappingLookup<F>),
```